bigtable-client-core-parent/bigtable-client-core/src/main/resources/com/google/cloud/bigtable/config/bigtable-version.properties

```diff
@@ -1,2 +1,7 @@
 bigtable.version=${project.version}
 
+# These are stable or supported version.
+stable.versions=1.11,1.10,1.9,1.8,1.7,1.6,1.5
+
+# These are obsolete versions.
+obsolete.versions=1.4,1.3,1.2,1.1
```

Comment:
new line?

Reply:
The new line is being removed by google plugin only for this file(`.properties`). I think I may have to skip this new line.